actor-to-actor/manifest.yaml

```diff
@@ -0,0 +1,14 @@
+---
+actors:
+  - ./pinger/target/wasm32-unknown-unknown/debug/pinger_s.wasm
+  - ./ponger/target/wasm32-unknown-unknown/debug/ponger_s.wasm
+capabilities:    
+    - image_ref: wasmcloud.azurecr.io/httpserver:0.12.1    
+links:    
+  # the ponger needs no link definition because it doesn't 
+  # use the HTTP server provider
+  - actor: ${PINGER_ACTOR}
+    contract_id: "wasmcloud:httpserver"
+    provider_id: "VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M"  
+    values:
+        PORT: 8080
```

Comment:
Because of this link, I'd love to see a `Makefile` included with this example (and denoted in the example README) where you can run this as-is with a `make run` step. A `make build` step could be useful as well to build and sign both actors.

The `make run` step would just need to mention `wash` is required as a prereq (which it already is) and then it could `wash claims inspect` the pinger actor and store that module value in the `PINGER_ACTOR` environment variable.

Essentially what I wanted to avoid with these examples is someone coming along to take a look at it and needing to debug why it's not working at all when they simply load the manifest.

Reply:
Done.